cfme/utils/appliance/__init__.py

```diff
@@ -1740,6 +1740,26 @@ def configuration_details(self):
             server id, server zone id)``
         """
         try:
+            servers = self.rest_api.collections.servers.all
```

Comment:
I would rather use ``list(self.rest_api.collections.servers)`` to make sure the data is up-to-date (``.all`` will not reload the data).

Reply:
Right but it's in a cached property anyway, so we are only doing it once anyway at the very beginning of the run